flink-connector/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+repositories {
+  mavenCentral()
+}
+
+val flinkVersion: String = libs.versions.flink.get()
+val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
+
+dependencies {
+  implementation(project(":api"))
+  implementation(project(":common"))
+  implementation(project(":core"))
+  implementation(project(":clients:client-java"))
```

Comment:
could you add a issue to track using `client-java-runtime` to build Flink connector?

Reply:
@FANNG1 Done. https://github.com/datastrato/gravitino/issues/3514